source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,234 @@
+relevance score for each result. For more information on relevance
+scores, see the :ref:`Text Search <sorts-crud-text-search>` section in
+our Sort Results guide.  
+
+Specify Options
+~~~~~~~~~~~~~~~
+
+You can include ``TextSearchOptions`` as the second parameter of the
+``Filters.text()`` method to specify text search options such as case
+sensitivity. By default, text searches run without case sensitivity
+which means the search matches lowercase and uppercase values.
+
+To specify a case sensitive search, use the following snippet:
+
+.. code-block:: java
+
+   TextSearchOptions options = new TextSearchOptions().caseSensitive(true);
+   Bson filter = Filters.text("SomeText", options);
+
+See the following API documentation for more information:
+
+- :java-core-api:`Filters.text() <com/mongodb/client/model/Filters.html#text(java.lang.String)>`
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text by a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass a term as a string to the ``Filters.text()`` method to specify the
+term in your text search. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "fast":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin termExample
+   :end-before: end termExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] }
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+To match multiple terms in your text search, separate each term
+with spaces in the ``Filters.text()`` builder method. The builder method
+returns the text search query as a ``Bson`` instance. When you pass
+this to the ``find()`` method, it returns documents that match any of
+the terms.
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+terms "fate" or "7":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin multipleTermExample
+   :end-before: end multipleTermExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass a phrase with **escaped quotes** to the ``Filters.text()`` method to
+specify the phrase in your text search. Escaped quotes are double quote
+characters preceded by a backslash character. If you don't add escaped
+quotes around the phrase, the method runs a :ref:`term search
```

Comment:
If "the method" is "Filters.text()", it specifies the search and doesn't run it right? If you're referring to find(), you'll need to specify that somewhere.